sdks/csharp/adtIngestFunctionSample.cs

```diff
@@ -34,21 +34,26 @@ public void Run([EventGridTrigger] EventGridEvent eventGridEvent, ILogger log)
             {
                 //Authenticate with Digital Twins
                 // <ManagedIdentityCredential>
-                ManagedIdentityCredential cred = new ManagedIdentityCredential("https://digitaltwins.azure.net");
+                var cred = new ManagedIdentityCredential("https://digitaltwins.azure.net");
```

Comment:
The type is obvious based on the right-hand side of the equals sign, so `var` is generally preferred here.